ArcGISRuntimeSDKQt_CppSamples/Maps/ConfigureBasemapStyleLanguage/ConfigureBasemapStyleLanguage.qml

```diff
@@ -0,0 +1,133 @@
+            // Set the focus on MapView to initially enable keyboard navigation
+            forceActiveFocus();
+        }
+    }
+
+    // Declare the C++ instance which creates the map etc. and supply the view
+    ConfigureBasemapStyleLanguageSample {
+        id: model
+        mapView: view
+    }
+
+    Rectangle {
+        id: rectangle
+        anchors {
+            right: parent.right
+            top: parent.top
+            margins: 15
+        }
+        width: column.width
+        height: column.height
+        color: Qt.rgba(0, 0, 0, 0.5)
+        border {
+            color: "black"
+            width: 1
+        }
+        opacity: 0.9
+
+        Column {
+            id: column
+            padding: 10
+            spacing: 5
+            width: 300
+
+            Column {
+                id:languageStrategy
+                spacing: 5
+
+                Text {
+                    text: "Set Language Strategy:"
+                    font.bold: true
+                    color: "white"
+                }
+
+                Row {
+                    RadioButton {
+                        id: globalButton
+                        onCheckedChanged: {
+                            setNewBasemap();
+                            localButton.checked = !globalButton.checked
+                        }
+                        enabled: comboBox.currentText == "none"
+                    }
+
+                    Text {
+                        id: globalText
+                        text: "Global"
+                        height: globalButton.height
+                        verticalAlignment: Text.AlignVCenter
+                        color: "white"
+                    }
+                }
+
+                Row {
+                    RadioButton {
+                        id: localButton
+                        checked: true
+                        onCheckedChanged: {
+                            setNewBasemap();
+                            globalButton.checked = !localButton.checked
+                        }
+                        enabled: comboBox.currentText == "none"
+                    }
+                    Text {
+                        id: localText
+                        text: "Local"
+                        height: localButton.height
+                        verticalAlignment: Text.AlignVCenter
+                        color: "white"
+                    }
+                }
+
+            }
+
+            Column {
+                id:specificLanguage
+
+                Text {
+                    text: "Set Specific Language:"
+                    font.bold: true
+                    color: "white"
+                }
+
+                ComboBox {
+                    id: comboBox
+                    model: ["none" , "Bulgarian", "Greek", "Turkish"]
+                    onCurrentTextChanged: setNewBasemap()
+                }
+            }
+        }
+    }
+
+    function setNewBasemap() {
```

Comment:
seems unnecessary to have the JS function wrapping the C++ function - why not just directly call the C++ function from the combobox itself?